README.md

```diff
@@ -3,17 +3,33 @@
 ## Get Qrypto
 Chome Web Store: https://chrome.google.com/webstore/detail/qrypto/hdmjdgjbehedbnjmljikggbmmbnbmlnd
 
-## Connecting Qrypto to your Web Dapp
-Connect to qrypto by calling 
-window.postMessage({ message: { type: 'CONNECT_QRYPTO' }}, '*')
+### Connecting Qrypto to your Web Dapp
+Connect to qrypto by calling `window.postMessage({ message: { type: 'CONNECT_QRYPTO' }}, '*')`
 
-This will populate the window.qrypto.account object in your webpage.
+This will populate the `window.qrypto.account` object in your webpage.
```

Comment:
they dont have to add the event listener first,  without the event listener they can still access window.qrypto.account, they just won't get notified when it changes(we will still change if the user logs in/out). similarly user doesn't need to log into qrypto first for the dapp to access window.qrypto.account, it will just return { loggedIn: false}

Reply:
👍 cool makes sense to have that in the separate section then